datastream-kafka-connector/src/test/java/com/linkedin/datastream/connectors/kafka/mirrormaker/TestKafkaTopicPartitionInfo.java

```diff
@@ -34,7 +34,7 @@
  * Tests for TopicPartitionStats diag command
  */
 @Test
-public class TestKafkaTopicPartitionStats extends BaseKafkaZkTest {
+public class TestKafkaTopicPartitionInfo extends BaseKafkaZkTest {
```

Comment:
what is the reason behind changing the class name?

Reply:
It was done by the refactoring tool. Reverting the change.